### ISSUE TITLE
Instructions for command line tool

### ISSUE THREAD

**Issue by dwhacks** [jim17/memtype#8]
Are there any instructions or plans to make instructions to use the command line tool??

**Comment by oyzzo** [comment]
Hi @dwhacks,
there were on the MemTypeGettingStarted.pdf under documents directory. We decided to stop using (and delete) the command line tool in favor of the new GUI tool with the Python library as it is much simpler to use and better done.

As we deleted the command line tool we also erased any reference to it in the documentation. The older version of the documentation with the command line tool instructions can be found here:
https://github.com/jim17/memtype/blob/a689661507a3e6efe56879e68b845eec3923c7d8/documents/MemTypeGettingStarted.pdf

For the newer GUI tool and python library take a look here:
https://github.com/oyzzo/MemTypeTool
